src/main/python/vyu/tracker.py

```diff
@@ -1,14 +1,32 @@
 from contextlib import contextmanager
+import imageio
+import numpy as np
+
+from vyu.image import image2position
+from vyu.timer import Timer
 
 
 class EyeTracker(object):
 
-    def __init__(self):
+    def __init__(self, camera='<video0>'):
         self.transform_matrix = np.eye(2)
         self.transform_bias = np.zeros(2)
+        self.reader = imageio.get_reader(camera)
 
-    def wait_for_fixation(self, loc, patience=0.2):
-        raise NotImplementedError
+    def wait_for_fixation(self, target_area, patience=0.2):
+        timer = Timer()
+        for frame in self.reader:
+            centroid = self.image2screen(image2position(frame))
+            if centroid in target_area:
+                if timer.start_or_check(patience):
+                    return
+            else:
+                timer.clear()
+
+    def image2screen(self, img_coords):
+        screen_coords = np.dot(self.transform_matrix, img_coords)
```

Comment:
How does this work? Are the `img_coords` a `(2, 2)` array? If so, do they define the eye position on the screen according to the camera?

Reply:
They are a (2,) array and they define the eye position on the screen.